nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -547,21 +560,32 @@ in {
             ''}
 
             ln -sf ${cfg.package}/apps ${cfg.home}/
+            rm -rf ${cfg.home}/nix-apps
+            mkdir -p ${cfg.home}/nix-apps
```

Comment:
Can you explain why you're using this points to `home` rather than `datadir`?

Reply:
Because store-apps is there. To me (personally) it did not make sense to put the apps in the datadir because they are not really data (to me). My Nextcloud is in an ephemeral container so with datadir I can wipe everything except the data on every update. Where do you think the apps should go @Ma27 ?

Reply:
Fair enough :+1: